### PR TITLE
Bump docker-thrift version to 0.9.3

### PR DESCRIPTION
**Pull request by schnipseljagd**
depends on: https://github.com/ahawkins/docker-thrift/pull/1 to be merged.

### CODE REVIEW

**Comment by yosifkit** [comment]
That new commit doesn't seem to exist :wink:.  Also, feel free to drop `0.9.2` if you no longer plan on supporting it; the library file here controls rebuilds and the "supported tags" list on the Docker Hub, but the image will still be available on the Docker Hub since old tags are never deleted.

**Comment by schnipseljagd** [comment]
The new commit isn't merged yet. It is a pull request as well :-)

**Comment by yosifkit** [comment]
Well, I am extra observant today :stuck_out_tongue:, thanks. Be sure to ping us here once it is merged, otherwise we might not see it.

File: library/thrift [hunk 1]
```diff
@@ -2,4 +2,5 @@
 
 0.9: git://github.com/ahawkins/docker-thrift@61c3478ab828d3e610f192b442ac2a7221749c47 0.9
 0.9.2: git://github.com/ahawkins/docker-thrift@61c3478ab828d3e610f192b442ac2a7221749c47 0.9
+0.9.3: git://github.com/ahawkins/docker-thrift@23d022c7b84399794349bf4ba7f5568020911f1d 0.9
 latest: git://github.com/ahawkins/docker-thrift@61c3478ab828d3e610f192b442ac2a7221749c47 0.9
```
Comment (tianon) [inline library/thrift]: PR is merged!  Ready to update all the commit hashes here and remove `0.9.2`, or would you prefer someone else take over from here? :smile:

**Pull request closed without merging**